update.js

```diff
@@ -1,18 +1,16 @@
-
 const NavBar: FC = () => {
     const [pageSwtichButtons, setPageSwitchButtons] = useState(
       InitialPageSwitchButtons
     );
-    const [currentButton, setCurrentButton] = useState(
-      InitialPageSwitchButtons[0]
-    );
+dfsdfsd
   
     return (
         <div className="d-flex flex-row align-items-center navbar-desktop-right-content">
           {pageSwtichButtons.map((button) => {
-                      alt=""
                       className="navbar-page-switch-button-icon"
+                      kjkjnkjn
```

Comment:
<div><div>This is another PR comment<br/><br/></div><div><img align="left" src="https://picsum.photos/200/200" width="100" height="100" title="My Pic"><b>Name</b> <sub>Just Now</sub><br/>Good Catch! I will go ahead and change this now. For now, we are not worried about endian, We will change it after. Thanks for the suggestion. Do you have any recommendations for os specific task that we may need to change further? I've been thinking about shifting some of our codebases to cloud services Also Is there any alternative to google Map API for showing maps and nearest places?</div></div>'

Reply:
This is another comment

Reply:
This is reply comment